torchdata/datapipes/iter/load/online.py

```diff
@@ -95,6 +95,9 @@ def __iter__(self) -> Iterator[Tuple[str, StreamWrapper]]:
     def __len__(self) -> int:
         return len(self.source_datapipe)
 
+def _extract_gdrive_api_response(content: str) -> Optional[str]:
+    match = re.search("<title>Google Drive - (?P<api_response>.+?)</title>", content)
+    return match["api_response"] if match is not None else None
```

Comment:
Could you please fix the lint Error? Basically, we need two empty lines between functions.